clip.py

```diff
@@ -61,7 +59,7 @@ class Clip():
                          5: "RECORDING"}
 
     def __init__(self, audio_file=None, name='',
-                 volume=1, frame_offset=0, beat_offset=0.0, beat_diviser=1):
+                 volume=1, frame_offset=0, beat_offset=0.0, beat_diviser=1, route_out=0, mute_group=0):
```

Comment:
can you rename route_out to output ? and set default value to 'MAIN'.

I would like to reference output by name without _L or _R suffix and store this in song file (sbs) with output key (with default value : 'MAIN')

So a clip with output set as 'DELAY' will use output ports : DELAY_L and DELAY_R.

There is no Mono / stereo feature in superboucle, so for compatibility for future version, I would like to store clip output as 'DELAY' instead as 'DELAY_R' and 'DELAY_L' with 2 key : output_1, output_2....